docs/releases/2.2.txt

```diff
@@ -475,6 +475,9 @@ Miscellaneous
 * Providing an integer in the ``key`` argument of the :meth:`.cache.delete` or
   :meth:`.cache.get` now raises :exc:`ValueError`.
 
+* Django 2.2.0 changed the behaviour of ``ngettext`` translations for plural
+  forms. Support for different plural equations was added in 2.2.12.
+
```

Comment:
Maybe:
```
* Plural equations for some languages are changed, because the latest versions
  from Transifex are incorporated.
  
  .. note::

    The ability to handle ``.po`` files containing different plural equations
    for the same language was added in Django 2.2.12.
```

@claudep What do you think?